notifications_api_common/viewsets.py

```diff
@@ -162,46 +169,39 @@ def notify(
         # build the content of the notification
         message = self.construct_message(data, instance=instance)
 
+        self.send_notification.delay(status_code, message)
+
+    @staticmethod
+    @celery.current_app.task(bind=True)
+    def send_notification(
+        task,
+        status_code: int,
+        message: dict,
+    ) -> None:
```

Comment:
why is this even a staticmethod and not a regular celery task in `tasks.py` that gets autodiscovered?